atsign-foundation/noports
# Update GUI to show relevant SSHNPParams properties

**- What I did**

1. removed `SSHNPParams` properties not relevant for the Home Screen.
2. update connection screen to show `SSHNPParams` properties that the user need to edit and rename field name to a more UI friendly name.
3. Set the appropriate `SSHNPPartialParams` properties to default SSHNP properties.
4. Refactor the delete config file code to use the `SSHNPParams.deletefile()` method.

closes #419 
5. **- How I did it**

**- How to verify it**
Run the sshnp_gui app and view home and new connection screens.
You should return to the Home Screen if you add or edit a config file and the home screen should show the config files you created or edited. 

**- Description for the changelog**


Update the sshnp_gui app to display more relevant content to users and keep app up to date with the recent changes to the sshnoports package.
